cmd/rekor-server/app/serve.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/sigstore/rekor-tiles/pkg/signerverifier"
 	"github.com/sigstore/rekor-tiles/pkg/tessera"
 	"github.com/sigstore/sigstore/pkg/signature"
+	tesseraLib "github.com/transparency-dev/trillian-tessera"
```

Comment:
Should this, by convention, be `tesseralib`?

Reply:
Now removed.